moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -493,6 +496,15 @@ const NewProjectMap = ({
             // hides feature selecting and drawing layers when when component is a signal
             return null;
           }
+          if (
+            drawLines === true &&
```

Comment:
do we need to check for strict equality here (and in the reverse case?)—seeing this implies to me that `drawLines ` could be something other than `true` or `false`

Reply:
drawLines is initially null and I'm making it do double duty in a way by assuming drawLines as null means no component is selected

I could add another state variable "componentSelected", keeping drawlines as only true or false. Then it would be if drawlines is true and componentSelected is true, return null etc.

Reply:
ah, got it! it's obvi now that i looked at the code, but can you just add a comment here that mentions if `drawLines` is `null`, this ensures no map drawing is possible?